test/k8resource/deployment.yaml

```diff
@@ -0,0 +1,325 @@
+# Copyright (C) 2020, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: verrazzano-operator
+  namespace: verrazzano-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This is an elaborate list of permissions, duplicated from the Helm chart. For the purpose of integ tests, my suggestion would be to cut this down to a single broad permission.

The set of permissions in the Helm chart is indirectly tested by the acceptance tests when they run.

Reply:
fixed